ec2/ec2-auto-recovery.yaml

```diff
@@ -139,6 +145,13 @@ Parameters:
     Description: 'Optional comma-delimited list of IAM managed policy ARNs to attach to the instance''s IAM role'
     Type: String
     Default: ''
+  RootVolumeSize:
```

Comment:
The new parameter is missing in the documentation?

Reply:
we don't have a documentation per parameter in this project?

Reply:
Sorry, mixed that up.